src/test/java/games/strategy/triplea/delegate/WW2V3_42_Test.java

```diff
@@ -94,6 +96,7 @@ public void testBombAndAttackEmptyTerritory() {
 
   @Test
   public void testLingeringSeaUnitsJoinBattle() throws Exception {
+    ProData.initializeSimulation(new ProAI("one","two"), m_data, germans(m_data));
```

Comment:
Hang on @ron-murhammer , didn't you ask me to use the AI to handle the case of two different defenseless units supporting each other and thus becoming non-defenseless? I never thought it was necessary and still don't, although I do see where you are coming from - would be difficult to track down in the unlikely event of it ever happening.

As for why here, pure pragmatism. I put it in and the test logic worked while it didn't work without it. I think it works because it's the first test fired which uses it.